src/site-type/html.php

```diff
@@ -170,7 +171,8 @@ private function configure_site_files() {
 		$filter[]               = $this->site_data['site_type'];
 		$site_docker            = new Site_HTML_Docker();
 		$docker_compose_content = $site_docker->generate_docker_compose_yml( $filter );
-		$default_conf_content   = $default_conf_content = \EE\Utils\mustache_render( SITE_TEMPLATE_ROOT . '/config/nginx/default.conf.mustache', [ 'server_name' => $this->site_data['site_url'] ] );
+		$default_conf_content   = \EE\Utils\mustache_render( SITE_TEMPLATE_ROOT . '/config/nginx/main.conf.mustache', [ 'server_name' => $this->site_data['site_url'] ] );
+		$custom_conf_content    = \EE\Utils\mustache_render( SITE_TEMPLATE_ROOT . '/config/nginx/custom.conf.mustache', [] );
```

Comment:
Why are you mustache rendering this file for nothing? Why not use `file_get_contents()`? If there is some plan of adding something later on then you can add mustache rendering at that time.